src/main/java/frc/robot/subsystems/elevator/ElevatorIO.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.subsystems.elevator;
+
+import edu.wpi.first.units.Distance;
+import edu.wpi.first.units.MutableMeasure;
+
+public interface ElevatorIO {
+    ElevatorInputs inputs = new ElevatorInputs();
+
+    void setPower(double power);
+
+    void setHeight(MutableMeasure<Distance> height);
+
+    void resetEncoder(boolean isBottom);
```

Comment:
Talon FX can reset its encoder automatically when a limit switch is triggered, so this should not be in the IO, and be implemented only in the sim IO.

Reply:
b267b0f7dd5a16e6ec5933f940d73637ea505cf5